packages/node/src/integrations/tracing/nest/sentry-nest-instrumentation.ts

```diff
@@ -16,7 +16,9 @@ const supportedVersions = ['>=8.0.0 <11'];
 /**
  * Custom instrumentation for nestjs.
  *
- * This hooks into the @Injectable decorator, which is applied on class middleware, interceptors and guards.
+ * This hooks into
+ * 1. @Injectable decorator, which is applied on class middleware, interceptors and guards.
+ * 2. @Catch decorator, which is applied on exception filters.
```

Comment:
This is left over I think

Reply:
no I forgot to update that comment when I added the exception filter tracing so that's fine